mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -95,14 +99,13 @@ public boolean hasService(String serviceName) {
 
   @SuppressWarnings("unchecked") //
   private <T> T findService(String serviceName) {
-    if (!isDestroyed()) {
-      if (SERVICE_NAME.equals(serviceName)) return (T) this;
+    assertNotDead();
```

Comment:
but currently it throws IllegalArgumentException because of the existing null check in getService